definitions/private-law/json/CaseEvent.json

```diff
@@ -672,7 +672,7 @@
     "Name": "Send and reply to messages",
     "Description": "Send and reply to messages",
     "DisplayOrder": 17,
-    "PreConditionState(s)": "SUBMITTED_PAID;CASE_ISSUE;GATE_KEEPING;AWAITING_RESUBMISSION_TO_HMCTS;CASE_WITHDRAWN",
+    "PreConditionState(s)": "SUBMITTED_PAID;CASE_ISSUE;GATE_KEEPING;AWAITING_RESUBMISSION_TO_HMCTS;CASE_WITHDRAWN;CASE_HEARING",
```

Comment:
Should we do the same as -   "[STATE]!=\"AWAITING_SUBMISSION_TO_HMCTS\" AND [STATE]!=\"DELETED\""? Is this possible? or the list will outgrow us.